### PR TITLE
deps(drivers): Bump versions of chromedriver and geckodriver

### PR DESCRIPTION
**Pull request by abraham**
~Updating chromedriver to get access to Shadow DOM v1 for testing https://github.com/angular/angular/pull/19059.~

### CODE REVIEW

**Comment by abraham** [comment]
I've fixed several of the tests but I'm too unfamiliar with this area to go about fixing the rest of the tests.

**Comment by cnishina** [comment]
ChromeDriver and GeckoDriver actually download the latest. The config.json is out of date and parts of them should be removed / cleaned up.

**Comment by cnishina** [comment]
I am also unfamiliar with e2e_spec/*.ts files. Are these fixes to existing tests that fail?

**Comment by abraham** [comment]
Here is the previous test run with the failures. https://circleci.com/gh/angular/webdriver-manager/165

A couple were formatting.

The others as far as I can tell were checking the API response from Selenium which appears to be returning a different success object than it use to. I'm don't know much about it but based on this doc and the tests were wrong.

https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#responses

File: e2e_spec/server_spec.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ describe('sever smoke tests', () => {
       let logs = '';
       resp.on('data', (chunk) => logs += chunk);
       resp.on('end', () => {
-        expect(logs).toContain('"state":"success"');
+        expect(logs).toContain('"status":0');
```
Comment (cnishina) [inline e2e_spec/server_spec.ts]: Could you explain this? Or list the log message in a comment.

File: config.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "webdriverVersions": {
     "selenium": "2.53.1",
-    "chromedriver": "2.27",
-    "geckodriver": "v0.13.0",
+    "chromedriver": "2.32",
```
Comment (cnishina) [inline config.json]: Sure? I think these get initialized but I believe they are always replaced. This section should be removed soon.

**Comment by cnishina** [comment]
This might require a rebase. Thanks!

**Comment by abraham** [comment]
Since `config.json` doesn't do anything anymore and they syntax fixes were merged already I don't think there is much left to merge.

**Pull request closed without merging**